MAINTAINERS

```diff
@@ -4,25 +4,10 @@
   "issues": "https://tickets.puppetlabs.com/browse/TK",
   "internal_list": "https://groups.google.com/a/puppet.com/forum/?hl=en#!forum/discuss-trapperkeeper-maintainers",
   "people": [
-    {
-      "github": "aperiodic",
-      "email": "dan.lidral-porter@puppet.com",
-      "name": "Dan Lidral-Porter"
-    },
     {
       "github": "lindboe",
```

Comment:
Congrats @lindboe you now own tk-status :)

Reply:
finally, i have been waiting for this day